Userland/Libraries/LibPDF/Parser.cpp

```diff
@@ -113,21 +119,25 @@ bool Parser::parse_header()
     return true;
 }
 
-bool Parser::initialize_linearization_dict()
+Parser::LinearizationResult Parser::initialize_linearization_dict()
 {
     // parse_header() is called immediately before this, so we are at the right location
     auto dict_value = m_document->resolve(parse_indirect_value());
     if (!dict_value || !dict_value.is_object())
-        return false;
+        return LinearizationResult::NotLinearized;
```

Comment:
I think this should be an `Error`. We're right after the header, so there should definitely be an indirect object of some kind here. Is this not true? (it's been a while since I worked on pdfs so I'm not certain)

Reply:
Yeah, I was wondering the same thing, but I couldn't find anything specific. I would mark it as an error, anyway.